web/src/inbox_ui.js

```diff
@@ -952,9 +955,4 @@ export function initialize() {
             unread_ops.mark_stream_as_read(stream_id);
         }
     });
-
-    $("body").on("focus", ".inbox-row, .inbox-header", (e) => {
-        const $elt = $(e.currentTarget);
-        center_focus_if_offscreen($elt);
-    });
```

Comment:
The previous code probably isn't the right mechanism for this, but how does this handle live-update from new messages coming in jumping the view on you so that you click the row next to the one you wanted?